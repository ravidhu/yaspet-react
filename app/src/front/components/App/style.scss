div.yaspet {

  width: 80%;
  margin: 0 auto;
  display: flex;
  flex-wrap: wrap;
  justify-content: space-around;

  div {

    flex: 1 auto;

    h1 {
      text-align: center;
    }

    h2 {
      text-align: center;
      font-style: italic;
      font-weight: 100;
    }

  }

}
