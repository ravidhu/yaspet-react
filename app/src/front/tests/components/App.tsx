import * as React from 'react';
import App from '../../components/App/App';
import * as renderer from 'react-test-renderer';

test('Link changes the class when hovered', () => {

    const component = renderer.create(<App />)

    let tree = component.toJSON();

    expect(tree).toMatchSnapshot();

});
