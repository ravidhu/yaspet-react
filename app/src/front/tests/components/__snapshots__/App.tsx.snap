// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link changes the class when hovered 1`] = `
<div
  className="yaspet"
>
  <div>
    <h1>
      Hello YASPET ! 
      <br />
       
      <small>
        React Edition
      </small>
    </h1>
    <h2>
      YASPET is a starter pack to create glorious Electron app
    </h2>
  </div>
  <div>
    <p>
      It's light like a breeze, it only come with :
    </p>
    <ul>
      <li>
        Electron 13
      </li>
      <li>
        Typescript 4
      </li>
      <li>
        React 17
      </li>
    </ul>
    <p>
      And Tooling for development :
    </p>
    <ul>
      <li>
        Webpack to bundle all front code and hot reload front during development.
      </li>
      <li>
        Nodemon to reload the main process during development.
      </li>
      <li>
        Jest for testing.
      </li>
    </ul>
  </div>
</div>
`;
